import_3dm/read3dm.py

```diff
@@ -22,10 +22,44 @@
 
 import os.path
 import bpy
-import rhino3dm as r3d
+
+def install_dependencies():
+    try:
+        try:
+            import pip
+        except:
+            from subprocess import call
+            print("Installing pip... "),
+            res = call("{} -m ensurepip".format(bpy.app.binary_path_python))
```

Comment:
Does this work for paths with spaces too? Or does binary_path_python already escape everything necessary?

Reply:
`binary_path_python` should handle escape everything, I think. It should be system specific, and handled by Blender.